### DISCUSSION TITLE
Release mode

### DISCUSSION THREAD

**Discussion by lessabruno** [fleaflet/flutter_map#1433] [Q&A] [answered]
Hi !! My flutter_map works great on debug mode, but when I build with realease mode, the tiles get all grey.

The markers still works... and I can pan the map (grey screen) aswell... I can see the marker moving when I pan.

Tested only on Android. 3 differente phones, with the same thing happening. 

Do anybody knows how to solve it ?

![grey_screen](https://user-images.githubusercontent.com/44441485/213277223-a5d953d1-86d7-432c-a10a-ba6f8e98da35.png)

**Comment by JaffaKetchup** [comment, marked as answer]
Hi @lessabruno,
Have you added the INTERNET permission to the manifest as shown in the Additional Setup instructions?

**Comment by lessabruno** [comment]
Hi !!! Thanks a lot for the quick answer !

I had the permission in another version of the code, so I didn´t thought about it. 

Sorry for the silly question, and thanks again.
